src/styles/components/_c-select.scss

```diff
@@ -1,16 +1,22 @@
 .c-select {
     height: 100%;
```

Comment:
Border of the navbar is hidden by the select: Possible hacks:  padding-bottom: 1px; or height:99% 😝